plugins/azure-devops-backend/README.md

```diff
@@ -23,9 +23,9 @@ Configuration Details:
 - `AZURE_TOKEN` environment variable must be set to a [Personal Access Token](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=preview-page) with read access to both Code and Build
 - `organization` is your Azure DevOps Services (cloud) Organization name or for Azure DevOps Server (on-premise) this will be your Collection name
 
-#### Multi Organization
+#### Multi Organization & Service Principles
 
-To support cases where you have multiple Azure DevOps organizations you will want to make sure to configure them in the `integrations.azure` section of your `app-config.yaml` as detailed in the [Azure DevOps Locations](https://backstage.io/docs/integrations/azure/locations) documentation.
+To support cases where you have multiple Azure DevOps organizations and/or you want to use a Service Principle you will want to make sure to configure them in the `integrations.azure` section of your `app-config.yaml` as detailed in the [Azure DevOps Locations](https://backstage.io/docs/integrations/azure/locations) documentation.
```

Comment:
```To use service-principals, leave out the static token, and it will use the matching credentials section by hostname in integration.azure```

Reply:
Reading that would make me thing you could not have the `azureDevOps.token` at all which isn't true as it's still  required. Did you just leave it blank? I'm fine with adding this just want to get it right 👍

Reply:
🤔  just doing some testing and I swear in the past if you just left out required config it would fail to start or the `yarn backstage-cli config:check` command would fail

```yaml
azureDevOps:
  host: dev.azure.com
  organization: my-company
```

Should fail as it's required here: 

https://github.com/backstage/backstage/blob/ee0b56c0a6f42b0157b3456dae8b832bbc5d4f3b/plugins/azure-devops-backend/config.d.ts#L30

If it was `token?: string` then yes, you could just leave it entirely out. But something is going on with the validation so 🤷